### PR TITLE
Issue #1 return to list after update and remember previous filters

### PR DESCRIPTION
**Pull request by rogerroelofs**
This takes the user back to the list when creating / updating a shelter.  I also added a DataTables parameter so the UI remembers table state when returning to the list.

### CODE REVIEW

File: app/views/shared/_table.html.erb [hunk 1]
```diff
@@ -21,9 +21,10 @@
...
 <script>
-window.onload = function() {
-  $('#data-table').DataTable({
-    "order": [[ 1, "desc" ]]
-  });
-};
+ window.onload = function() {
+   $('#data-table').DataTable({
+     "order": [[ 1, "desc" ]],
+     "bStateSave": true
```
Comment (tdooner) [inline app/views/shared/_table.html.erb]: Are you sure `bStateSave` exists? I couldn't find it documented.

Also, I could be reading this diff wrong, but it looks like you indented this javascript by 1 space.

Comment (tdooner) [reply app/views/shared/_table.html.erb]: I think it is just `stateSave` now:
https://datatables.net/reference/option/stateSave

Comment (rogerroelofs) [reply app/views/shared/_table.html.erb]: I'll check it out and update.  Thanks!

**Comment by rogerroelofs** [comment]
I updated to the new parameter name and double checked the spacing.

**Comment by tdooner** [comment]
Awesome. Let's :shipit:

**Pull request merged**